client/src/features/visualizer-threejs/store/tangle.ts

```diff
@@ -64,6 +65,11 @@ interface TangleState {
 
     updateBlockIdToAnimationPosition: (updatedPositions: Map<string, IBlockInitPosition>) => void;
     resetConfigState: () => void;
+
+    // Handle finalized blocks
```

Comment:
this comment is misleading because these are for accepted/confirmed blocks, not finalized